lib/validators/html-validator.js

```diff
@@ -7,7 +7,8 @@ const util = require('util');
 
 const ERROR_TEMPLATE =
   `Config file validation error for route '%s' - '%s':
-        * File(s) for HTML rendering not found: %s.`;
+        * File(s) for HTML rendering not found: %s.
+        * Retrieving frontmatter failed for file(s): %s.`;
```

Comment:
I would use a different word for "retrieving", like parsing or something like that.